user/views.py

```diff
@@ -0,0 +1,65 @@
+import json
+import traceback
+import bcrypt
+import jwt
+
+from django.views           import View
+from django.http            import JsonResponse
+from django.core.exceptions import ValidationError
+
+from kibon.settings         import SECRET_KEY
+from .models                import User
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            signup_user = User(
+                name         = data['name'],
+                birthdate    = data['birthdate'],
+                identifier   = data['identifier'],
+                password     = data['password'],
+                gender       = data['gender'],
+                phone_number = data['phone_number'],
+                email        = data['email']
+            )
+            signup_user.full_clean()
+            signup_user.password = bcrypt.hashpw(signup_user.password.encode('utf-8'), bcrypt.gensalt())
+            signup_user.password = signup_user.password.decode('utf-8')
```

Comment:
```suggestion
            signup_user.password = bcrypt.hashpw(signup_user.password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```
굳이 두 번 선언하지 않으셔도 괜찮습니다.